llvm/test/CodeGen/AMDGPU/pal-metadata-3.0-callable.ll

```diff
@@ -0,0 +1,290 @@
+; RUN: llc -mtriple=amdgcn--amdpal -mcpu=gfx1100 -verify-machineinstrs < %s | FileCheck %s
```

Comment:
No - auto-generating adds check lines for all the functions as well, which isn't necessary, so I'd prefer to keep this as a manual check.

Reply:
> Would testing other metadata fields like wgp_mode require separate files, or could this be extended to cover them?
It would require a separate test to check some of the other fields. I'm reasonably happy that the checking for those in pal-metadata-3.0.ll is sufficient.